src/js/chartAuthoring.js

```diff
@@ -26,18 +26,56 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             },
             dataEntryPanel: {
                 type: "floe.chartAuthoring.dataEntryPanel",
-                createOnEvent: "onTemplatesLoaded",
-                container: "{chartAuthoring}.container",
+                createOnEvent: "onPieChartReady",
```

Comment:
Is there a reason that the creation of the data entry panel is dependent on the creation of the pie chart? Shouldn't they be independently instantiated once their templates are ready?